Nodes/LOGGER_BUTTON/LOGGER_BUTTON.ino

```diff
@@ -0,0 +1,80 @@
+#include <CAN.h>
+#include <millisDelay.h>
+#include "src/can_conv/sbm_network_definition.h"
+
+can_obj_sbm_network_definition_h_t can_obj;
+
+millisDelay Button_Press; 
+bool Logger_Read = false;
+int buttonState = 0;
+
+const int buttonPin = 12;
+
+union MsgData
+{
+  uint64_t a;   // dbcc output uses uint64_t
+  uint8_t b[8]; // Adafruit CAN library wants uint8_t[8]
+};
+
+void setup() {
+  pinMode(buttonPin, INPUT);
+  
+  pinMode(PIN_CAN_STANDBY, OUTPUT);
+  digitalWrite(PIN_CAN_STANDBY, false); // turn off STANDBY
+  pinMode(PIN_CAN_BOOSTEN, OUTPUT);
+  digitalWrite(PIN_CAN_BOOSTEN, true); // turn on booster
+  
+  Serial.begin(115200);
+  CAN.begin(500000); // start the CAN bus at 500 kbps (kilo-bits/second)
+
+  Button_Press.start(50);
+}
+
+void loop() {
+  if(Button_Press.justFinished())
```

Comment:
You'll want to check this inside of another condition, otherwise you're just checking this on a timer.